buildsystem/versions.gradle

```diff
@@ -10,7 +10,7 @@ ext {
             // Runtime dependencies
             androidx_app_compat       : '1.1.0',
             androidx_card_view        : '1.0.0',
-            androidx_constraint_layout: '1.1.3',
+            androidx_constraint_layout: '2.0.0-beta4',
```

Comment:
I would normally be against using anything that is not at least a release candidate, but i suppose we don't use a lot of the newer API that they are adding so it should be safe, right?

Reply:
I have reverted to the older version, as we currently don't use motion layout